### DISCUSSION TITLE
Logging data to the console.

### DISCUSSION THREAD

**Discussion by brianraila** [Rahix/avr-hal#312] [Q&A] [answered]
Is this supported. 
Wanted to use this for debugging purposes.

**Comment by stappersg** [comment]
[Begin](http://www.catb.org/~esr/faqs/smart-questions.html) and please [close this issue](https://github.com/Rahix/avr-hal/issues/258#issuecomment-1105031993).

**Comment by Rahix** [comment, marked as answer]
@brianraila, here is example code: https://github.com/Rahix/avr-hal/blob/main/examples/arduino-uno/src/bin/uno-usart.rs

**Comment by stappersg** [comment]
[another way of logging](https://defmt.ferrous-systems.com/#intended-use)

**Comment by brianraila** [comment]
> @brianraila, here is example code: https://github.com/Rahix/avr-hal/blob/main/examples/arduino-uno/src/bin/uno-usart.rs

Thanks so much, this works!
